src/plugins/discourse/update_discourse_api_snapshots.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -eu
+
+snapshots_dir=src/plugins/discourse/snapshots
+test_instance_url="https://sourcecred-test.discourse.group"
+test_instance_username="credbot"
+
+if [ -z "${DISCOURSE_TEST_API_KEY:-}" ]; then
+  printf >&2 'Please set the DISCOURSE_TEST_API_KEY environment variable.\n'
+  printf >&2 'Contact the SourceCred maintainers to get the key.\n'
+  return 1
+fi
+
+toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)"
+cd "${toplevel}"
+
+fetch() {
+  url="${test_instance_url}${1}"
```

Comment:
style nit: We generally don’t brace positional parameters (though we
still do quote them, of course): suggest `$1` rather than `${1}`.